test/modules/layers/scatterplot-layer.spec.ts

```diff
@@ -6,6 +6,11 @@ import * as FIXTURES from 'deck.gl-test/data';
 
 const SIZE = 1;
 
+function getUniforms(layer: Layer) {
+  // @ts-ignore
+  return layer.getModels()[0]._uniformStore.uniformBlocks.get('scatterplot').uniforms;
```

Comment:
I believe you can do `model.shaderInputs.getUniformValues()`

Reply:
I figured it was better to check if the `uniformStore` as this is closer to the GPU